packages/web/src/components/notification/Notifications/FollowNotification.tsx

```diff
@@ -37,10 +37,10 @@ type FollowNotificationProps = {
 
 export const FollowNotification = (props: FollowNotificationProps) => {
   const { notification } = props
-  const { users, timeLabel, isRead } = notification
-  const [firstUser, ...otherUsers] = users
-  const otherUsersCount = otherUsers.length
-  const isMultiUser = users.length > 1
+  const { users, userIds, timeLabel, isRead } = notification
+  const [firstUser] = users
```

Comment:
This is cool - we don't need `otherUsers` anymore!

Reply:
Yeah realized that couldn't be relied on since we only pull 8 users max, so need the ids from the actual notif to know the true number of users

Reply:
Wait but should it be `const otherUsersCount = userIds.length - 1` below? Won't this double count the first user?

Reply:
Yup nice catch